nncf/quantization/algorithms/channel_alignment/algorithm.py

```diff
@@ -112,10 +113,23 @@ def filter_func(point: StatisticPoint) -> bool:
             assert len(tensor_collectors) == 1
             stat = tensor_collectors[0].get_statistics()
             if stat.min_values is None or stat.max_values is None:
+                nncf_logger.debug(
+                    f"Skipping channel alignment for pairs {conv_in.node_name}, {conv_out.node_name} "
+                    "because statistics were not collected for this pair."
+                )
                 continue
 
             conv_in_cont = ConvParamsContainer(conv_in, model, graph, self._backend_entity)
             conv_out_cont = ConvParamsContainer(conv_out, model, graph, self._backend_entity)
+            if (
+                conv_in_cont.dims.conv_weight_out_channels_dim is None
+                or conv_out_cont.dims.conv_weight_out_channels_dim is None
+            ):
+                nncf_logger.debug(
+                    f"Skipping channel alignment for pairs {conv_in.node_name}, {conv_out.node_name} "
+                    " because one of the node is 1D MatMul, 1D Matmuls are not supported by CA algortihm yet."
+                )
```

Comment:
Could you please explain this message? Is `conv_in` a Convolution or MatMul node?

Reply:
The condition is that at least one of the matched operation is a 1d MM, which means it has no output dimension. We can solve it by reshaping weights, but I suggest to keep it for future PRs.

Reply:
O, but `conv_in` or `conv_out` name for a MatMul node is confusing.